examples/with-sfcc/package.json

```
@@ -0,0 +1,27 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "commerce-sdk": "^2.8.0",
+    "next": "latest",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
```

```suggestion
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
```

Could you also verify that all other dependencies are up to date?